### PR TITLE
Refactor ChatOpenAI_ChatModels to include stopSequence parameter

### PR DESCRIPTION
**Pull request by definitelynotchirag**
- Stop Sequence added into OpenAI params

### CODE REVIEW

File: packages/components/nodes/chatmodels/ChatOpenAI/ChatOpenAI.ts [hunk 1]
```diff
@@ -111,6 +111,13 @@ class ChatOpenAI_ChatModels implements INode {
                 optional: true,
                 additionalParams: true
             },
+            {
+                label: 'Stop Sequence',
+                name: 'stopSequence',
+                type: 'string',
+                optional: true,
+                additionalParams: true
```
[the remark this answers is not in the document]
Comment (definitelynotchirag) [reply packages/components/nodes/chatmodels/ChatOpenAI/ChatOpenAI.ts:121]: Alright check my latest commit!

**Review by HenryHengZJ — APPROVED** [review]
thanks @definitelynotchirag !

**Comment by definitelynotchirag** [comment]
@HenryHengZJ can you please add the hacktoberfest tag, so it can get accepted as a PR for hacktoberfest?

**Comment by HenryHengZJ** [comment]
Is it too late now?

**Comment by definitelynotchirag** [comment]
@HenryHengZJ no it isn't, thank you!!!!

**Pull request merged**